content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -167,6 +167,39 @@ fail validation.
 				</ul>
 			</td>
 		</tr>
+		<tr>
+			<td style="white-space: nowrap">Probes or Lifecycle Hooks with Host field</td>
```

Comment:
make this shorter, and let it wrap

```suggestion
			<td>Host Probes / Lifecycle Hooks (v1.34+)</td>
```

Reply:
see https://deploy-preview-51498--kubernetes-io-main-staging.netlify.app/docs/concepts/security/pod-security-standards/ for how this appears

Reply:
good point! I did see the preview but missed this bit. fixed it